frontend/src/scenes/experiments/ExperimentView/components.tsx

```diff
@@ -274,13 +274,33 @@ export function NoResultsEmptyState(): JSX.Element {
         )
     }
 
-    // Non-400 errors are rendered as plain text
+    if (experimentResultCalculationError?.statusCode === 504) {
```

Comment:
this ought to be any 5XX ?

Reply:
a timeout should always be 504, to be handled by this specific message

for any other error we have a more generic message down below: `Experiment results could not be calculated`